develop-docs/sdk/data-model/event-payloads/contexts.mdx

```diff
@@ -778,6 +778,56 @@ envelope endpoint.
 }
 ```
 
+## Page Context
+
+Page context contains information about the page that the event occurred on.
+
+`full.url`
```

Comment:
It feels a bit overkill for me to store everything in separate fields. What's wrong with just having a `url` field?

Reply:
cc @cleptric had concerns about pii stripping etc!

Reply:
https://github.com/getsentry/sentry-javascript/pull/7667 for context. We imo should not emit the full URL.

Reply:
I updated this to be: `url`, `http.query` and `http.fragment`.

Reply:
This still feels like a mess because `url` sounds like it contains the full url.

How about we lean it on the `window.location` API, which I'd assume is something people can relate to?:

- `location.origin`
- `location.pathname`
- `location.search`
- `location.hash`

From that you can reconstruct the the entire URL (minus some PII redactions).